Makefile

```diff
@@ -0,0 +1,19 @@
+# This makefile uses only POSIX 'make' features, except for the
+# following widely implemented extensions -- the .PHONY target, which
+# marks targets that don't generate a file whose name is the same as the
+# target, and prerequisites that depend on glob expansion.
+
+.POSIX:
+.SUFFIXES:
+
+.DEFAULT: preview
+.IGNORE: preview
+.PHONY: preview
+preview:
+	swift package --disable-sandbox preview-documentation --target TSPL
+
+.PHONY: archive
+archive: .build/plugins/Swift-DocC/outputs/TSPL.doccarchive
+
+.build/plugins/Swift-DocC/outputs/TSPL.doccarchive: Sources/TSPL/TSPL.docc/*/*.md
```

Comment:
This should probably extend beyond `.md` files, e.g., images in the archive.

Reply:
Changed in 93b5be45.